Plugin/CleverTapUnity/Editor/CleverTapPostBuildProcessor.cs

```diff
@@ -72,9 +72,16 @@ public static void OnPostProcessBuild(BuildTarget target, string path)
 			proj.AddFrameworkToProject(projectTarget, "CoreLocation.framework", false);
 			proj.AddFrameworkToProject(projectTarget, "Security.framework", false);
 
-            // Add linker flags
-            proj.AddBuildProperty(projectTarget, "OTHER_LDFLAGS", "-ObjC");
+			// Add linker flags
+			proj.AddBuildProperty(projectTarget, "OTHER_LDFLAGS", "-ObjC");
 
+#if UNITY_2018_3_OR_NEWER
```

Comment:
@HarryHodgson What is being executed here that's Unity 2018.3 specific? Is it the calls to PBXProject?

In any case, can you add a comment to show why the directive is here.